<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>331</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);
color: rgb(117, 117, 117);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);
font: 9pt &quot;맑은 고딕&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>선수를 추가하세요</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>121</width>
       <height>106</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ss1">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="stat1">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ss2">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="stat2">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ss3">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="stat3">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ss4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;맑은 돋움&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="stat4">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="ss6">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ss5">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ss7">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ss8">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>스탯8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="stat7">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="stat6">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="stat5">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="stat8">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>181</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">
font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>이름 :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="name">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">
font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>소속팀 :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="team">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>포지션 :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="position">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>나이 :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="age">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>국적 :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="nation">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>키 :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="height">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>몸무게 :</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="weight">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;휴먼옛체&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;휴먼엑스포&quot;;</string>
     </property>
     <property name="text">
      <string>예상몸값  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="income">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;맑은 고딕&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>만원</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>311</width>
       <height>10</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>연령 : </string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>421</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1990</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>450</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>480</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 66, 66);

font: 9pt &quot;굴림&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>선수추가</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>이적료 : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 66, 66);

font: 9pt &quot;굴림&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>선수삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>430</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 66, 66);

font: 15pt &quot;굴림&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>추천</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="Valuelabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 13pt &quot;맑은 고딕&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>0 만원</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 209, 255);
color: rgb(0, 0, 0);

font: 11pt &quot;맑은 고딕&quot;;

</string>
    </property>
    <property name="text">
     <string>프로그램 종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 20pt &quot;Lucida Bright&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Football Scouter</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(166, 209, 255);
color: rgb(0, 0, 0);

font: 10pt &quot;굴림&quot;;
</string>
       </property>
       <item>
        <property name="text">
         <string>분류</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>이름</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>국적</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>소속팀</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>포지션</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;굴림&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;

font: 9pt &quot;굴림&quot;;
background-color: rgb(166, 209, 255);</string>
       </property>
       <property name="text">
        <string>검색</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>641</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 211);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
